### ISSUE TITLE
Web Version 2.0 Task list

### ISSUE THREAD

**Issue by devosama1** [citizenos/citizenos-web#447]
**What is done**
A wordpress instance is created, server is configured and its live for previewing and working on the dev environment. all changes are done as per this [document](https://docs.google.com/document/d/1Q0Fgpc9Rx6bfmxZjFEC9zMSMwC9bf7agVymTBCmYwYI/edit?pli=1#) and figma designs are ready and implemented.

**What needs to be done?**
As further design is being added to figma, there is no clear comment as to what is ready to be published for the new web that can be added to the website. Also there are alot of internal pages that needs a clarification as to which design to follow, the old or the new? 

Also some pages that seems to be ready but are being worked on by the designer has been paused until its ready to be previewed.

- Implementation of WPML and Crowdin for languages (Will add when first demo is cleared) 
- Futher addition of missing pages and content adjustments
- Creating shortcode for news section and stats section
- security hardening further, cleaning up and organising templates 
- Training session with any team member who can help with content
- Final integration will be plausible
- Monitoring tool for server and issues (Wp)

**Timeline**

- Project started 1st week of April 2023
- 3rd of April Access to AWS billing to setup AWS server for wordpress instance
- 13th of April purchase of Elementor Pro for template designing
- First demo on 10th of May

Maintenance, track time for it (Toggle) (1 hours a month basic updates, 2 hours a week )

**Comment by devosama1** [comment]
@oksks 
Estimate pre-launch estimation

- Homepage **Done**
- Projects **2 hours for any adjustments, and 3 days for new projects design to be implemented**
- Donate **20 mins for any adjustment**
- Vision **Done**
- News (and internal pages) **Done**
- About **10 mins for any internal updates needed**
- Get in involved **Done**
- Team **Done, just content need to be added**
- Contact **Done**
- legal **1 hour for content addition**
- Help Center (and internal pages) **2-3 hours for further template adjustment and testing across** and also **YES/NO button Plausible integration 1 day as per new discussion**
- Google Translate Button **Done**


**New work, Timeline section, which will become part of Projects (New Design) will take 2 days extra**


Template Naming Convention (Example Page) (All the blocks that gets approved or ready and tested for responsive gets loaded to the template page with the name, which you could reference and use the name to find the templates in the template import section) 1 full day, but might take more as more sections get ready. I will also have to reference back to figma too Here https://www.figma.com/file/tcsdpFuL39ZwGpE0LSOCAi/CitizenOS-Web-2.0?type=design&node-id=1%3A2&t=sGw8aRAUm3pLdQvP-1

**Comment by oksks** [comment]
Plus testing & bugfixing after the above list is done, I'd say 2 days? @devosama1

**Comment by devosama1** [comment]
@oksks Yes agreed :)

As i see major things 

- Projects new design implementation
- HC plausible integration
- New page design where vision, team and about is combined

Above might take 3-4 days of work, apart from this i guess any bug fixes or responsive adjustments needed can be taken care of as testing is done, and i can help with content as well.

But overall Except above I am finished with things.

**Comment by oksks** [comment]
Discussed the timelines further. 
Considering both me and @devosama1 work half-time and there's the whole Help Center and News section to copy-paste manually and new designs to get finalized and implemented + templated, realistically we aim for June 20 to launch & announce the web internally, then allow for any feedback to flow in and get implemented/fixed by June 22.

Public Web 2.0 launch can happen thereafter - after Midsummers', I guess? 
FYI @MeelikaH and @anettlinno.

**Comment by anettlinno** [comment]
> Discussed the timelines further. Considering both me and @devosama1 work half-time and there's the whole Help Center and News section to copy-paste manually and new designs to get finalized and implemented + templated, realistically we aim for June 20 to launch & announce the web internally, then allow for any feedback to flow in and get implemented/fixed by June 22.
> 
> Public Web 2.0 launch can happen thereafter - after Midsummers', I guess? FYI @MeelikaH and @anettlinno.

Sounds very good!! Thanks for the deadline @oksks and @devosama1 ! :)

**Comment by devosama1** [comment]
Due to AWS server conflicts and restrictions, I had finally managed to get in touch with them to lift the restrictions and adjusted the EC2 instance to make sure the code, 3rd party integrations and links work correctly and not produce CORS issues while translating with google translate. It took quite bit of time and it added delay to the project. This was also linked with other items that were currently being added and design adjusted, as these issues created a ripple effect on delivery time across all my task list. 

We successfully launched on 29th of July with domain running citizenos.com the wordpress instance.
